include/utils/point_locator_base.h

```diff
@@ -202,14 +220,25 @@ class PointLocatorBase : public ReferenceCountedObject<PointLocatorBase>
 
   /**
    * \p true if we will use a user-specified tolerance for locating
-   * the element.
+   * the element in an exhaustive search.
    */
   bool _use_close_to_point_tol;
 
   /**
    * The tolerance to use.
    */
   Real _close_to_point_tol;
+
+  /**
+   * \p true if we will use a user-specified tolerance for locating
+   * the element.
+   */
+  bool _use_contains_point_tol;
```

Comment:
Both new class members (but especially the boolean) should be initialized in the constructor so they don't have indeterminate values if the user never calls `set_contains_point_tol()`. I'm not sure if this is what's causing the unit tests to fail, but it's possible that your new functionality is being used by default because this bool is not properly initialized.

Reply:
Good call. :man_facepalming: Adding that change now...